source/PlayState.hx

```diff
@@ -2,15 +2,50 @@ package;
 
 import flixel.FlxSprite;
 import flixel.FlxState;
+import flixel.text.FlxText;
+import flixel.FlxG;
 
 class PlayState extends FlxState
 {
-	var hero:Hero;
+
+	var screensaverHero1:ScreensaverHero;
+	var screensaverHero2:ScreensaverHero;
+	var screensaverHero3:ScreensaverHero;
+	var screensaverHero4:ScreensaverHero;
+	var controllableHero:ControllableHero;
+
 	override public function create()
 	{	
-		hero = new Hero(300,300);
-		add(hero);
+
+		controllableHero = new ControllableHero(500, 400);
+		add(controllableHero);
+
+		screensaverHero1 = new ScreensaverHero(50, 0);
+		screensaverHero2 = new ScreensaverHero(50, 75);
+		screensaverHero3 = new ScreensaverHero(50, 150);
+		screensaverHero4 = new ScreensaverHero(50, 325);
```

Comment:
Loops are your friend here, something like:
```
for (i in 0...4) {
    add(new ScreensaverHero(i * 50, i * 50))
}
```

This gets the job done, which is a fine result when you're under time pressure, but it's not hard to see the pain that repetition brings. That, and it doesn't hurt to get more comfortable with the fundamentals of the Haxe language.

No action required here, but optionally swap to a loop if you want the practice.